manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml

```diff
@@ -44,6 +44,11 @@ spec:
         node-role.kubernetes.io/master: ""
       restartPolicy: Always
       tolerations:
+      - key: node.cloudprovider.kubernetes.io/uninitialized
+        effect: NoSchedule
+        value: "true"
+      - key: node.kubernetes.io/not-ready
+        effect: NoSchedule
```

Comment:
Is this documented somewhere? I wouldn't have expected this to be required from the enhancement. Would be good to find official word on this to make sure this is the effect of the external provider and not some pot luck or other change we aren't aware of

Reply:
The node is being marked as not ready because the CNI driver is not yet running. We need to make sure the CNI driver tolerates the unitialized taint. Then networking will come up as expected and we won't need to tolerate this not-ready taint on any of our pods.

This should be considered a blocker for this PR IMO

Reply:
i think the openshift-sdn component is already set to tolerate this, see https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/openshift-sdn/controller.yaml#L60

Reply:
I removed the not-ready taint, should be ready to go.